### DISCUSSION TITLE
Multiple Pages Bref AWS

### DISCUSSION THREAD

**Discussion by moedawood** [brefphp/bref#987] [Support] [answered]
Hi 

Does bref allow for multiple php pages 

Example 

User logs in -> Direct to register page-> Register page

All this running on aws lambda

**Comment by deleugpn** [comment, marked as answer]
Yes. Maybe the Bref for Web Apps might be a good place to start: https://bref.sh/docs/runtimes/http.html

**Reply by moedawood** [reply]
Okay went through it , Do you have a sample serverless.yml that caters for multiple pages ?

**Reply by mnapoli** [reply]
With the "Bref for Web Apps" runtime, all requests are sent to `index.php`. This is the pattern that most PHP frameworks follow today.

Routing based on multiple PHP files (i.e. without routing frameworks, e.g. `/index.php`, `/products.php`, `/list/users.php`, etc.) isn't supported. Maybe it's possible to run that, but it's not really how Bref works tbh so I think you would struggle a lot.

**Reply by moedawood** [reply]
Okay Perfect 

Thank you for the assistance
